test/e2e/knative_serving_test.go

```diff
@@ -59,12 +59,18 @@ func TestKnativeServing(t *testing.T) {
 			t.Fatalf("Found a custom-metrics API registered at the autoscaler")
 		}
 
-		// The list of deployments that are HA-ed
+		// Check the status of deployments in the knative serving namespace
 		for _, deployment := range []string{"controller", "autoscaler-hpa"} {
 			if err := test.CheckDeploymentScale(caCtx, knativeServing, deployment, haReplicas); err != nil {
 				t.Fatalf("Failed to verify default HA settings: %v", err)
 			}
 		}
+		// Check the status of deployments in the ingress namespace.
+		for _, deployment := range []string{"3scale-kourier-control, 3scale-kourier-gateway"} {
```

Comment:
This needs to be `for _, deployment := range []string{"3scale-kourier-control", "3scale-kourier-gateway"} {`  ;-)

Reply:
See the double quotes in different places.

Reply:
🤦

Reply:
Don't worry. That was my mistake in the code that you copied over :-D